packages/external-db-postgres/lib/sql_filter_transformer.js

```diff
@@ -245,6 +245,11 @@ class FilterParser {
         }]
     }
 
+    projection(projection) {
+        if (!projection) return '*'
```

Comment:
I'm making sure that the projection will not be empty when it's arriving here (in `schema_aware` layer)
I should remove this line?